projects/sequence_prediction/continuous_sequence/run_arima.R

```diff
@@ -4,11 +4,11 @@ library(forecast)
 library(TSPred)
 
 # set working directory here
-setwd('/Users/ycui/nta/nupic.research/projects/sequence_prediction/continuous_sequence')
+setwd('/Users/ycui/nta/htmresearch/projects/sequence_prediction/continuous_sequence')
```

Comment:
This might need to be changed?

Reply:
I did not validate experiments written in  R or matlab with this PR, only experiments written in python. I am not sure if those experiments still work or the software requirements. I will probably validate in a future PR.

Reply:
Was just thinking that `setwd('/Users/ycui/nta/...` will work on very few laptops 😄

Reply:
But fine to merge...

Reply:
How about this? 00a9eb1

It uses the current script location instead of hardcoded location